### PR TITLE
add Mentors schema

### PR DESCRIPTION
**Pull request by hercase**

### CODE REVIEW

File: src/schemas/mentor.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    {
+      title: 'Calendly',
+      name: 'calendly',
+      type: 'string',
+      validation: (Rule) => Rule.required(),
+    },
+    {
+      title: 'LinkedIn',
+      name: 'linkedin',
+      type: 'string',
```
Comment (martinbavio) [inline src/schemas/mentor.ts]: URL type maybe

File: src/schemas/mentor.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    },
+    {
+      title: 'Github',
+      name: 'github',
+      type: 'string',
```
Comment (martinbavio) [inline src/schemas/mentor.ts]: URL type too?

File: src/schemas/mentor.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    },
+    {
+      title: 'Temas',
+      name: 'temas',
+      type: 'array',
+      of: [{ type: 'block' }],
```
Comment (martinbavio) [inline src/schemas/mentor.ts]: Shouldn't it just be an array of strings?

**Review by martinbavio — COMMENTED** [review]
Some minor issues but other than that LGTM!

**Comment by hercase** [comment]
I updated the types and added the "topics" category to the scheme

**Comment by martinbavio** [comment]
@hercase LGTM! 🚀

**Pull request merged**